automation/jenkins/aws/manageUnits.sh

```diff
@@ -146,6 +146,21 @@ function main() {
 
         # Manage the stack if required
         if [[ -n "${DEPLOYMENT_MODE}" ]]; then
+          if [[ "${DEPLOYMENT_MODE}" == "${DEPLOYMENT_MODE_PLAN}" ]]; then
```

Comment:
It might be better to now use the deployment mode to specify this. Now that you can add new deployment modes with deployment profiles what if I wanted to plan deployment of dfferent mode?

Reply:
Dryrun now supported on manageUnits instead of relying on deployment mode. 

I've also added deployment mode, if present, to the plan branch name so you can see the effect of different modes independently and easily differentiate between them.